.github/workflows/CI.yml

```diff
@@ -58,11 +58,17 @@ jobs:
             os: macos-13
           - version: "1.6"
             os: windows-latest
+          # Performance regressions on Julia 1.11 on Windows. Currently not
+          # viable to run the test suite there.
+          - version: "1.11"
+            os: windows-latest
```

Comment:
I am disabling these for now. The performance regressions seem to be related to some of the integration tests that we are running here, rather than anything this package itself does. Hopefully these get resolved upstream. I'll open an issue to track re-enabling 1.11 on Windows once this has improved.